test/services/EmploymentServiceSpec.scala

```diff
@@ -65,4 +68,34 @@ class CreateEmploymentServiceSpec extends TestUtils {
     }
   }
 
+  "deleteEmployment" should {
+
+    "return Right" when {
+
+      "deleteEmployment connector call succeeds" in {
+        (mockDeleteEmploymentConnector.deleteEmployment(_: String, _: Int, _: String)(_: HeaderCarrier))
+          .expects(nino, taxYear, employmentId, *)
+          .returning(Future.successful(Right(())))
+
+        val result = createEmploymentService.deleteEmployment(nino, taxYear, employmentId)
+
+        await(result) mustBe Right(())
+      }
+    }
+
+    "return Left containing DesError" when {
+      "the deleteEmployment connector call fails" in {
+        val desError = DesErrorModel(500, DesErrorBodyModel("DES_CODE", "DES_REASON"))
```

Comment:
This 500 and the one on line 58 give magic number weak warnings, could be turned into INTERNAL_SERVER_ERROR's if you want to get rid of them?

Reply:
yep